src/cclib/parser/qchemparser.py

```diff
@@ -481,6 +483,12 @@ def extract(self, inputfile, line):
             if 'basis functions' in line:
                 if not hasattr(self, 'nbasis'):
                     self.set_attribute('nbasis', int(line.split()[-3]))
+                    # We can't display more MOs than there are basis
```

Comment:
The problem is the parser needs to keep track of how many MOs are going to be displayed. By default, this is all the occupied MOs and the first 5 virtual MOs. However, it isn't known how many MOs this could be until the number of electroons and the number of basis functions are encountered.

In the process of figuring out the display number, it may become larger than the number of MOs that are physically present. This would happen if there are fewer than 5 virtual MOs.

I've tried changing the comment to clarify this.